src/containers/location/providers/NoResults.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+
+import React from 'react';
+
+import styled from 'styled-components';
+import { faSearchLocation } from '@fortawesome/pro-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { isImmutable, Map } from 'immutable';
+import { Button, Typography } from 'lattice-ui-kit';
+import { DataUtils } from 'lattice-utils';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { getTextFnFromState } from '../../../utils/AppUtils';
+import { getDistanceBetweenCoords } from '../../../utils/DataUtils';
+import { PROPERTY_TYPES } from '../../../utils/constants/DataModelConstants';
+import { NO_RESULTS } from '../../../utils/constants/Labels';
+import { PROVIDERS, STATE } from '../../../utils/constants/StateConstants';
+import { getCoordinates } from '../../map/MapUtils';
+import { Centered, Row } from '../../styled';
+import { selectReferralAgency } from '../LocationsActions';
+
+const Instructions = styled(Typography)`
+  padding: 15px;
+  text-align: center;
+`;
+
+const { LOCATIONS } = STATE;
+const {
+  LAT,
+  LON,
+  REFERRAL_AGENCY_LOCATIONS,
+  SELECTED_OPTION
+} = PROVIDERS;
+
+const { getEntityKeyId, getPropertyValue } = DataUtils;
+
+const NoResults = () => {
+  const dispatch = useDispatch();
+  const getText = useSelector(getTextFnFromState);
+  const nearbyReferralAgencies = useSelector((store) => store.getIn([LOCATIONS, REFERRAL_AGENCY_LOCATIONS], Map()));
+  const selectedOption = useSelector((store) => store.getIn([LOCATIONS, SELECTED_OPTION]));
+
+  const renderRR = (rr :Map) => {
+    const name = getPropertyValue(rr, [PROPERTY_TYPES.FACILITY_NAME, 0]);
+    const handleViewProfile = () => {
+      dispatch(selectReferralAgency(rr));
+    };
+
+    let { lat, lon } = selectedOption;
```

Comment:
just curious, if `selectedOption` is an immutable, will this object destructuring break things?

Reply:
I don't think it does, but It might make more sense to do this:
```
        let lat;
        let lon;
        if (isImmutable(selectedOption)) {
          lat = selectedOption.get(LAT);
          lon = selectedOption.get(LON);
        }
        else {
          lat = selectedOption.lat;
          lon = selectedOption.lon;
        }
```

Reply:
FANTASTIC!